hugegraph-pd/hg-pd-client/pom.xml

```diff
@@ -34,7 +34,7 @@
         <dependency>
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
-            <version>1.18.20</version>
+            <version>1.18.30</version>
```

Comment:
Shall we use a unified version property for it?

Reply:
+1